reviewable-request-close-deferred-payment.x

```diff
@@ -0,0 +1,27 @@
+
+
+%#include "xdr/types.h"
+%#include "xdr/operation-payment.h"
+
+
+namespace stellar
+{
+struct CloseDeferredPaymentRequest {
+    uint64 deferredPaymentID;
+
+    AccountID destination;
+    BalanceID destinationBalance;
```

Comment:
Why both destination and destination balance instead of how it is done in payment?